src/app/@ansyn/overlays/components/overlay-overview/overlay-overview.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="overlay-overview"
+	 [class.show]="showFlag"
+>
+	<div class="flex-parent"
+	>
+		<main>
```

Comment:
💯

Reply:
I am also removing the "flex-parent" div, and moving its style def to its parent. I needed this additional div just because I used the "hidden" attribute on the parent, which does not work together with "display: flex". but I don't use it any more...